src/main/scala/com/codacy/client/bitbucket/v2/service/RepositoryServices.scala

```diff
@@ -10,8 +10,8 @@ class RepositoryServices(client: BitbucketClient) {
    * Gets the list of the user's repositories. Private repositories only appear on this list
    * if the caller is authenticated and is authorized to view the repository.
    */
-  def getRepositories(username: String): RequestResponse[Seq[Repository]] = {
-    client.executePaginated(Request(s"https://bitbucket.org/api/2.0/repositories/$username", classOf[Seq[Repository]]))
+  def getRepositories(owner: String): RequestResponse[Seq[Repository]] = {
```

Comment:
What is this owner? Is it the ownerUUID? ownerId? ownerUsername?

Reply:
This can be the ownerId for a user account and a owner username for the teams